src/server/status/index.js

```diff
@@ -1,11 +1,16 @@
 import ServerStatus from './server_status';
 import wrapAuthConfig from './wrap_auth_config';
+import { Metrics } from './metrics';
 
 export default function (kbnServer, server, config) {
   kbnServer.status = new ServerStatus(kbnServer.server);
 
   if (server.plugins['even-better']) {
-    kbnServer.mixin(require('./metrics').collectMetrics);
+    const metrics = new Metrics(config, server);
+
+    server.plugins['even-better'].monitor.on('ops', event => {
+      metrics.capture(event).then(data => { kbnServer.metrics = data; });
```

Comment:
Is there a simplification you see it making? I fear that adding a getter for `latest` will become problematic when we begin doing roll-ups for CPU percentage, etc as we need to capture since the last call.